pkg/descheduler/apis/config/types_loadaware.go

```diff
@@ -65,12 +66,18 @@ type LowNodeLoadArgs struct {
 	// A resource consumption above (resp. below) this window is considered as overutilization (resp. underutilization).
 	UseDeviationThresholds bool
 
-	// HighThresholds defines the target usage threshold of resources
+	// HighThresholds defines the target usage threshold of node resources
 	HighThresholds ResourceThresholds
 
-	// LowThresholds defines the low usage threshold of resources
+	// LowThresholds defines the low usage threshold of node resources
 	LowThresholds ResourceThresholds
 
+	// ProdHighThresholds defines the target usage threshold of Prod resources
+	ProdHighThresholds ResourceThresholds
+
+	// ProdLowThresholds defines the low usage threshold of Prod resources
+	ProdLowThresholds ResourceThresholds
+
```

Comment:
After adding the new threshold, is the goal of the descheduling to satisfy both requirements at the same time? In other words, are they in an AND relationship?

Reply:
ps: If `ProdLowThresholds` is nil, set as `MinResourcePercentage` or `MaxResourcePercentage` in `newThreshold` function. This makes sure all nodes are under-utilized so nothing will do by this plugin. And the same logic is used to `LowThreshold`, too.